rationale.md

```diff
@@ -37,25 +55,26 @@ Purely for illustration purposes, not intended to represent actual releases:
 
 - `opentelemetry-api` 1.0.0
   - Contains APIs for tracing, baggage, propagators, context
-- `opentelemetry-tracing` 1.0.0
-  - Contains the tracing SDK
 - `opentelemetry-sdk` 1.0.0
   - Contains SDK components for tracing, baggage, propagators, and context
 
 ##### Contains the following experimental packages
 
-- `opentelemetry-api-metrics` 0.x.y
-  - Contains the EXPERIMENTAL API for metrics. There are no stability guarantees.
-- `opentelemetry-metrics` 0.x.y
-  - Contains the EXPERIMENTAL SDK for metrics. There are no stability guarantees.
+- `opentelemetry-api` 0.x.y-b0
+  - Contains the EXPERIMENTAL API for metrics plus other unstable features. There are no stability guarantees.
+- `opentelemetry-metrics` 0.x.y-b0
```

Comment:
was the name of the package here supposed to be `opentelemetry-sdk` instead of metrics? I thought we agreed on not creating a separate package name.